common/src/main/java/org/conscrypt/OpenSSLExtendedSessionImpl.java

```diff
@@ -70,17 +66,6 @@ AbstractOpenSSLSession getDelegate() {
         };
     }
 
-    /* @Override */
-    @SuppressWarnings("MissingOverride") // For Android backward-compatibility.
-    public List<SNIServerName> getRequestedServerNames() {
```

Comment:
I've set the return type to List, and it looks like that allows me implement the function using reflection.